### DISCUSSION TITLE
Submission tiers for new name

### DISCUSSION THREAD

**Discussion by em9** [seq-code/registry#140] [Q&A] [answered]
Hi SeqCode folks, I heard there is a two-tier system for submitting new names but I am still a little unclear on how it functions. Just to clarify, can I propose a name prior to my publication being submitted? Thanks!

**Comment by lmrodriguezr** [comment, marked as answer]
Hello @em9 

Yes, you can submit your names in parallel with the manuscript preparation (or during the review process). This is known in the SeqCode as "Path 1", and it's the preferred method. Once your names are ready to be reviewed, you can simply submit them and we will help check their correctness, thereby reducing potential errors in the eventual published manuscript. We recommend all names be "Endorsed" by the curator team before submitting the manuscript for review. Once the manuscript has been published, you can then "Notify" the Registry, in order for the names to be considered validly published.

To get started, visit https://seqco.de/

I hope this answers your question, but please feel free to send us any follow-up questions.

Best wishes,
Miguel.
